py/selenium/webdriver/remote/switch_to.py

```diff
@@ -53,7 +53,7 @@ def alert(self) -> Alert:
                 alert = driver.switch_to.alert
         """
         alert = Alert(self._driver)
-        alert.text
+        _ = alert.text
```

Comment:
Why are we doing this?

Reply:
just to indicate the value is never used; will revert this one